libraries/c_sdk/aws/shadow/test/system/aws_iot_tests_shadow_system.c

```diff
@@ -422,6 +432,24 @@ static void _updateGetDeleteBlocking( IotMqttQos_t qos )
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
```

Comment:
If this module is already using the `_` character to notate `static` functions, this is acceptable.  However be aware that some coding standards may not allow this.